test/rating.js

```diff
@@ -49,7 +49,7 @@ describe('A track in Google Music', function () {
   describe('when \'thumbs up\'-ed', function () {
     browserUtils.execute(function resetRating () {
       // DEV: Warning this will skip to next track
-      window.googleMusic.rating.toggleThumbsDown();
+      window.googleMusic.rating.toggleThumbsUp();
```

Comment:
It still isn't working, I might just rewrite the test so I know what the hell is going on :+1: